src/components/BookList.jsx

```diff
@@ -0,0 +1,17 @@
+import { useSelector } from 'react-redux';
+import AddBook from './AddBook';
+import Book from './Book';
+
+export default function BookList() {
+  const bookList = useSelector((state) => state.booklist.value);
+
+  return (
+    <section className="books-container-section">
+      {bookList.map((book) => (
+        <Book key={book.id} title={book.title} author={book.author} id={book.id} />
+      ))}
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/233161921-6ef0e08a-4dc7-44af-8ba3-ccf1b0f026c6.png)

**[OPTIONAL]**

- [ ] It would be very nice if you add **npm start** command in the `Usage` section and the **npm run build** command in the `Deployment `section to make your readme useful and descriptive 👍